packages/fondue/src/components/RichTextEditor/Plugins/EmojiPlugin/EmojiPicker/EmojiPickerContent.tsx

```diff
@@ -0,0 +1,164 @@
+                            {i18n.categories[categoryId]}
+                        </div>
+                        <div
+                            className="tw-relative tw-flex tw-flex-wrap"
+                            style={{ height: section.getRows().length * buttonSize.value }}
+                        >
+                            {isCategoryVisible(categoryId) &&
+                                section
+                                    .getRows()
+                                    .map((row: GridRow, index) => (
+                                        <RowOfButtons
+                                            key={index}
+                                            emojiLibrary={emojiLibrary}
+                                            row={row}
+                                            onSelectEmoji={onSelectEmoji}
+                                            onMouseOver={onMouseOver}
+                                        />
+                                    ))}
+                        </div>
+                    </div>
+                );
+            });
+    }, [emojiLibrary, getRowWidth, i18n.categories, isCategoryVisible, onSelectEmoji, onMouseOver, settings]);
+
+    const SearchList = useCallback(() => {
+        return (
+            <div data-id="search" style={{ width: getRowWidth }}>
+                <div className="tw-sticky tw--top-px tw-z-[1] tw-bg-white/90 tw-p-1 tw-backdrop-blur-[4px]">
+                    {i18n.searchResult}
+                </div>
+                <div className="tw-relative tw-flex tw-flex-wrap">
+                    {searchResult.map((emoji: Emoji, index: number) => (
+                        <Button
+                            key={emoji.id}
+                            index={index}
+                            emoji={emojiLibrary.getEmoji(emoji.id)}
+                            onSelect={onSelectEmoji}
+                            onMouseOver={onMouseOver}
+                        />
+                    ))}
+                </div>
+            </div>
+        );
+    }, [emojiLibrary, getRowWidth, i18n.searchResult, searchResult, onSelectEmoji, onMouseOver]);
+
+    return (
+        <div
+            className={cn(
+                'tw-h-full tw-min-h-[50%] tw-overflow-y-auto tw-overflow-x-hidden tw-px-3',
+                '[&::-webkit-scrollbar]:tw-w-4',
+                '[&::-webkit-scrollbar-button]:tw-hidden [&::-webkit-scrollbar-button]:tw-h-0 [&::-webkit-scrollbar-button]:tw-w-0',
+                '[&::-webkit-scrollbar-thumb]:hover:tw-bg-[#f3f4f6]',
+                '[&::-webkit-scrollbar-thumb]:tw-min-h-[65px] [&::-webkit-scrollbar-thumb]:tw-rounded-2xl [&::-webkit-scrollbar-thumb]:tw-border-solid [&::-webkit-scrollbar-thumb]:tw-border-4 [&::-webkit-scrollbar-thumb]:tw-border-white',
+                '[&::-webkit-scrollbar-track]:tw-border-0',
+            )}
```

Comment:
```suggestion
            className={merge([
                'tw-h-full tw-min-h-[50%] tw-overflow-y-auto tw-overflow-x-hidden tw-px-3',
                '[&::-webkit-scrollbar]:tw-w-4',
                '[&::-webkit-scrollbar-button]:tw-hidden [&::-webkit-scrollbar-button]:tw-h-0 [&::-webkit-scrollbar-button]:tw-w-0',
                '[&::-webkit-scrollbar-thumb]:hover:tw-bg-[#f3f4f6]',
                '[&::-webkit-scrollbar-thumb]:tw-min-h-[65px] [&::-webkit-scrollbar-thumb]:tw-rounded-2xl [&::-webkit-scrollbar-thumb]:tw-border-solid [&::-webkit-scrollbar-thumb]:tw-border-4 [&::-webkit-scrollbar-thumb]:tw-border-white',
                '[&::-webkit-scrollbar-track]:tw-border-0',
            ])}
```

Wouldn't introduce `cn` anywhere use the `merge`, or here it can be just a one liner... 

Imo we can get rid of `udecode/cn` completely

Reply:
Fixed these